docs/content/flink/procedures.md

```diff
@@ -210,7 +195,7 @@ All available procedures are listed below.
          -- and if there is no match,<br/> 
          -- insert the order from<br/>
          -- the source table<br/>
-         CALL sys.merge_into('default.T', '', '', 'default.S', 'T.id=S.order_id', '', 'price=T.price+20', '', '*')
```

Comment:
Do not introduce this usage in documentation.

`''` is ugly, and is just a workaround for us.